src/core/models/filter.js

```diff
@@ -23,6 +24,27 @@ export default class Filter {
     }
   }
 
+  /**
+   * Whether this filter is a range filter.
+   *
+   * @param {Filter} filter
+   * @returns {boolean}
+   */
+  isRangeFilter () {
+    const filterKey = this.getFilterKey();
+    if (!filterKey) {
+      return false;
+    }
+    const matchers = Object.keys(this[filterKey]);
+    const rangeMatchers = new Set([
```

Comment:
do we want to make this every time we call `isRangeFilter`? could we store this as a constant somewhere?

Reply:
ah yeah definitely